go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/CMSgov/bcda-ssas-app
+
+go 1.15
+
+require (
```

Comment:
Want to doublecheck, these deps match up with what used to exist in the Gopkg.lock file, right?

Reply:
Okay, I will check again.

Reply:
Offline conversation decided to manage dependencies based on the last used commit per `dep` and update each package separately with necessary code changes.  This method will assure we are using the same exact package to the commit during this change over to Go Modules.  Will update and submit changes before resolving conversation.